pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -387,3 +372,12 @@ func (d *kubeDockerClient) holdHijackedConnection(tty bool, inputStream io.Reade
 	}
 	return nil
 }
+
+// ImageNotFoundError is the error returned by InspectImage when image not found.
+type ImageNotFoundError struct {
```

Comment:
@yujuhong suggests that we should use private type here. I'll change this to private in the final cleanup PR. Just mark here. :)